src/python/.git_archival.txt

```diff
@@ -0,0 +1,3 @@
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe:tags=true,match=*[0-9]*)$
```

Comment:
It's about which tags will be considered for the version names. E.g., a tag `foo` wouldn't be used. This can be restricted further if it's decided that we stick to vYYYY.MM.DD, but I think this will do for the moment.

Reply:
This is about the tag to create a GitHub release, right? I guess we want all GitHub releases to be properly versioned and then made available on PyPI (TODO) and Zenodo -- if there is a release that isn't tagged with `*[0-9]*`, then it was probably a mistake on our side?

Reply:
This is about any tags that may exist in the repository, whether they are created by a GitHub release or by any other means. This will also be used when archiving any revision that is not a release. The tag that will be used as the basis of the version number will be the one closest up the tree that matches the given pattern.